### DISCUSSION TITLE
php版本

### DISCUSSION THREAD

**Discussion by 1270827935** [krahets/hello-algo#928] [General]
现阶段虽然php使用率低，但是我还是希望能出个php版的，在算法这行，都没有看到php版的，leetcode也是，萌新小白很无奈，虽然java有一定基础，但是现在行情不好，对于我们下游刚毕业一两年的萌新来说，后面找工作，无奈被迫转了php，希望大大有空能出个php的，球球球啦！祝大大身体健康，事事顺利，永远不死！

**Comment by krahets** [comment]
Hi ，谢谢建议！本书中的各语言代码是由多位贡献者维护的，PHP 暂时还没有小伙伴来贡献。如果你有兴趣，可以参考 #15 工作流，为这本书添加 PHP 版本代码。
